doc/changes/changes_3.2.1.md

```diff
@@ -0,0 +1,11 @@
+# Test Database Builder 3.2.1, released 2021-XX-XX
```

Comment:
Version 3.2.0 was not yet released. (indicated by the XX in the release date)
So simply add it there.